showhide.js

```diff
@@ -3,7 +3,7 @@
     var elems = $('*[showhide], *[onclick*=wp_showhide]'),
       $elem, id, text, settings, visible;
 
-    for (var i = 0; i < elems.length; i++) {
+    for (var j = 0; j < elems.length; i++) {

```

Comment:
Oh ok. Let me know if you have any questions about github and git. If I have some time I'll spin up a wordpress instance and test it myself, but I think the change on the site you linked to is referring to an older version of this plugin.